### ISSUE TITLE
Error with 'python -m autogpt' command. Please set your OpenAI API key in .env or as an environment variable. You can get your key from https://beta.openai.com/account/api-keys

### ISSUE THREAD

**Issue by Streete** [Significant-Gravitas/Auto-GPT#1902]
### Duplicates

- [X] I have searched the existing issues

### Steps to reproduce 🕹

Installed the 'stable' version of the program
I run 'python -m autogpt' command and comes up with an error.

![Screenshot 2023-04-16 183147](https://user-images.githubusercontent.com/130889399/232320050-2b495403-55e9-4d43-b588-e53172eba533.jpg)

I have paid Chat GPT and Open AI API accounts.
For Chat GPT I have access to version 4
For Open AI API I do not have access to version 4, I am on the version before this.

### Current behavior 😯

Error message ;Please set your OpenAI API key in .env or as an environment variable.
You can get your key from https://beta.openai.com/account/api-keys'

### Expected behavior 🤔

Should load the program as to start commands

### Your prompt 📝

```yaml
python -m autogpt```

**Comment by Qoyyuum** [comment]
Hi @Streete please set up your Open AI Key into your .env file and try again.

**Comment by CAIOHSR** [comment]
I have the same problem. I already changed the .env data several times and it still gives the same error

**Comment by sjnt1** [comment]
This was why i started reinstalling everything. I fixed it by using my old env file that i had a backup of and it worked for a while. Makes me think its something with the env file, make sure its env file and not template 


but now nothing works after i tried reinstalling with cloning the repo with git

**Comment by D-ignite** [comment]
Did you rename your .env file from .env.template = .env?

**Comment by CAIOHSR** [comment]
yes!

and inside the .env, it's exactly like that with the keys:

PINECODE_API_KEY=(NUMBER KEY)
PINECODE_ENV=(NUMBER KEY)
OPENAI_API_KEY=(NUMBER KEY)
ELEVEN_LABS_API_KEY=(NUMBER KEY)
ELEVENLABS_VOICE_1_ID=(NUMBER KEY)

**Comment by xx255q** [comment]
You should take a screenshot of how you currently have it setup in the file (maybe blur the actually key)

**Comment by Streete** [comment]
> Hi @Streete please set up your Open AI Key into your .env file and try again.

Hi @Qoyyuum, I have tried this about 4 times during the day.  Even to the point of generating new API keys and using those instead.  Still have the same problem.  Also, I have deleted and re-installed the auto-gpt program and still facing the same issue.

**Comment by Streete** [comment]
Hi @xx255q 

This is the section where I posted my open API Key

![image](https://user-images.githubusercontent.com/130889399/232323616-b0cb96e9-c451-44db-8080-007761674024.png)

The rest I have left untouched.

**Comment by Streete** [comment]
> Did you rename your .env file from .env.template = .env?

Hi @D-ignite 
Yes, I renamed the file to remove the template part.

**Comment by Qoyyuum** [comment]
Hi @Streete 

1. upgrade to a paid account: https://platform.openai.com/account/billing/overview
2. ChatGPT+ != paid account. 
3. add a payment method: https://platform.openai.com/account/billing/payment-methods (optional) set my usage limits: https://platform.openai.com/account/billing/limits
4. create a new API key: https://platform.openai.com/account/api-keys (old API key did not work, i guess it was not update with my payment information automatically)
5. replace the value for the `.env` files's `OPENAI_API_KEY`.

**Comment by CAIOHSR** [comment]
> 
these are my mistakes

<img width="115" alt="image" src="https://user-images.githubusercontent.com/121357597/232324475-e989cf08-c9da-431f-b241-17a3fd6ddcfd.png">

<img width="547" alt="image" src="https://user-images.githubusercontent.com/121357597/232324616-eac7d02a-8faa-48e6-9eb5-3bfa2d707053.png">

![image](https://user-images.githubusercontent.com/121357597/232324797-81e4b586-c0bf-443c-b653-2c9aba159e99.png)

**Comment by Qoyyuum** [comment]
> > 
> these are my mistakes
> 
> <img width="115" alt="image" src="https://user-images.githubusercontent.com/121357597/232324475-e989cf08-c9da-431f-b241-17a3fd6ddcfd.png">
> 
> <img width="547" alt="image" src="https://user-images.githubusercontent.com/121357597/232324616-eac7d02a-8faa-48e6-9eb5-3bfa2d707053.png">
> 
> ![image](https://user-images.githubusercontent.com/121357597/232324797-81e4b586-c0bf-443c-b653-2c9aba159e99.png)
> 

Your file is still a `.env.template`. Please do the following in your terminal

`cp .env.template .env`

This will copy and paste the `.env.template` file into `.env` file properly.

**Comment by CAIOHSR** [comment]
Thank you bro ! Solved

**Comment by Elcrypto87** [comment]
For me it does not work with cp. I get the following message. What am I doing wrong?

**Comment by Elcrypto87** [comment]
![image](https://user-images.githubusercontent.com/119629925/232326588-4ff46af5-66cb-4f9a-9632-6039721ae8fa.png)

**Comment by Streete** [comment]
@Qoyyuum 

Thank you for your input, it seems I had the same trouble as @CAIOHSR and that I hadnt renamed the file correctly.

**Comment by robowr** [comment]
If you are using cmd use copy comand instead of cp, in powershell use cp or copy it takes both commands


> ![image](https://user-images.githubusercontent.com/119629925/232326588-4ff46af5-66cb-4f9a-9632-6039721ae8fa.png)

If you are using cmd use copy comand instead of cp, in powershell use cp or copy it takes both commands

**Comment by Streete** [comment]
I used 'ren .env.template .env' which seem to work for my windows CMD portal

**Comment by robowr** [comment]
Yeah, ren or rename will do the trick. 😉 Best luck with your project
